.travis.yml

```diff
@@ -1,7 +1,22 @@
+sudo: true
 language: python
 python:
  - "2.7"
+addons:
+  apt:
+    sources:
+      - mongodb-3.0-precise
+    packages:
+      - mongodb-org-server
+before_install:
+  - sudo add-apt-repository ppa:fake-juju/master -y
+  - sudo apt-get update -q
+  - sudo apt-get install -y fake-juju
+env:
+  global:
+    - JUJU_MONGOD=/usr/bin/mongod
 install:
- - pip install twisted fixtures PyYAML testtools pyOpenSSL mocker
+ - pip install twisted==16.0.0 fixtures PyYAML testtools pyOpenSSL mocker fakejuju git+https://github.com/testing-cabal/testresources.git#egg=testresources
```

Comment:
I'd move these requirements to a requirements-travis.txt file, so they're a bit easier to maitain